tests/bounded.rs

```diff
@@ -130,6 +130,44 @@ fn close() {
     assert_eq!(q.pop(), Err(PopError::Closed));
 }
 
+#[test]
+fn get_head() {
+    let mut q = ConcurrentQueue::bounded(2);
+    assert_eq!(q.push(10), Ok(()));
+
+    assert!(q.get_head().is_ok());
+    assert_eq!(q.get_head(), Ok(10));
```

Comment:
These two lines are redundant, since the second line will check if the result of `get_head` is `Ok`.